src/Microsoft.DotNet.CodeFormatting/Analyzers/ExplicitThisAnalyzer.cs

```diff
@@ -15,17 +16,16 @@ namespace Microsoft.DotNet.CodeFormatting.Analyzers
     [DiagnosticAnalyzer(LanguageNames.CSharp)]
     public class ExplicitThisAnalyzer : DiagnosticAnalyzer
     {
-        internal static DiagnosticDescriptor rule = new DiagnosticDescriptor("DNS0001",
```

Comment:
Make the string available to the fixer so it doesn't have to repeat the hard-coded strings.